identity/manifest.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "chrome.identity example",
+  "version": "1",
+  "manifest_version": 2,
+  "applications": {
+    "gecko": { "id": "identity.example@mozilla.org" }
```

Comment:
Ah right. Could you add a comment about that somewhere in the code, that's a useful peice of information to anyone using the API and will stop someone removing that in the future 😄